dali/operators/reader/reader_op_test.cc

```diff
@@ -491,8 +491,10 @@ class FileReaderTest : public DALITest {
   }
 
   void TearDown() override {
-    for (const auto &path : filepaths_)
-      std::remove(path.data());
+    for (const auto &path : filepaths_) {
+        int ret = std::remove(path.data());
+        DALI_ENFORCE(ret == 0, "Failed to remove the file: ", path, ". Error code: ", ret);
```

Comment:
DALI_ENFORCE or GTEST ASSERT?

Reply:
IIRC, GTest's macros (`ASSERT` etc...) could be used only in a test body. This code is in the test fixture tear down. Unless the usage possibilities for checkers changed?

Reply:
Make sense.